src/avatar/lib/avatar.scss

```diff
@@ -0,0 +1,22 @@
+.tbds-avatar {
+  --border-radius: 3px;
+  --size: 3rem;
+
+  border-radius: var(--border-radius);
+  display: inline-block;
+  height: var(--size);
+  object-fit: cover;
```

Comment:
regarding CSS variables and object-fit: Should we describe somewhere in the package what browsers support tbds?

Reply:
We should, and do!

> tbds supports:
>
>    - Latest versions of Chrome, Firefox, and Safari
>    - Edge 15+


https://github.com/thoughtbot/design-system#browser-support